Integrations/UrlScan/URLScan.py

```diff
@@ -272,13 +272,6 @@ def format_results(uuid):
             dbot_score['Score'] = 0
             dbot_score['Type'] = 'url'
             human_readable['Malicious'] = 'Benign'
-    if 'url' in scan_meta['processors']['gsb']['data'] is None:
```

Comment:
we are sure it is no longer returned?

Reply:
@amshamah419 said that it was a breaking change in the API over the weekend but I can confirm if you'd like me to doublecheck